packages/paste-core/components/corner-ornament/src/CornerOrnamentElement.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, safelySpreadBoxProps } from "@twilio-paste/box";
+import { useUID } from "@twilio-paste/uid-library";
+import * as React from "react";
+
+import { useCornerOrnamentContext } from "./CornerOrnamentContext";
+import { BadgeBottomEndPath, DotBottomEndPath } from "./Masks";
+import { CornerOrnamentElementProps, CornerOrnamentPosition, CornerOrnamentType } from "./types";
+
+export const CornerOrnamentElement = React.forwardRef<HTMLDivElement, CornerOrnamentElementProps>(
+  ({ padding, element = "CORNER_ORNAMENT_ELEMENT", ...props }, ref) => {
+    const id = useUID();
+    const { cornerOrnamentType, position, size } = useCornerOrnamentContext();
+
+    const ClipPathMapping: Record<CornerOrnamentType, Record<CornerOrnamentPosition, string>> = {
+      badge: {
+        bottom_end: BadgeBottomEndPath,
+        top_end: BadgeBottomEndPath,
+      },
+      dot: {
+        bottom_end: DotBottomEndPath,
+        top_end: DotBottomEndPath,
+      },
+      icon: {
+        bottom_end: BadgeBottomEndPath,
+        top_end: BadgeBottomEndPath,
+      },
+      avatar: {
+        bottom_end: BadgeBottomEndPath,
+        top_end: BadgeBottomEndPath,
+      },
+    };
+
+    return (
+      <Box
+        {...safelySpreadBoxProps(props)}
+        style={{
+          clipPath: `url("#${id}")`,
```

Comment:
have to use a unique ID on clip paths and svgs otherwise using multiple on the same page will cause issues

Reply:
question: where does `url` come from?